UPGRADE.md

```diff
@@ -24,6 +24,17 @@
 * There are scalar and return typehints introduced in the codebase. Please introduce these in your codebase for classes 
   that implement Sylius' interfaces or extend Sylius' classes.
 
+* **Only when on Symfony ^3.3.8:**
+  From this Symfony version, the custom autoloader is not needed anymore and therefore removed in favor of the Composer 
+  autoloader. Apply the following changes (reference: https://github.com/Sylius/Sylius/pull/8340):
+
+  * Remove `app/autoload.php`
+  * Change autoload path in `bin/console`: replace 'vendor' with 'app'
```

Comment:
I think you meant the other way around? :)

Reply:
Wow, haha, indeed :)